cloud/pkg/policycontroller/policycontroller.go

```diff
@@ -54,6 +56,7 @@ func NewAccessRoleControllerManager(ctx context.Context, kubeCfg *rest.Config) (
 }
 
 func setupControllers(ctx context.Context, mgr manager.Manager) error {
+	ctrlruntimelog.SetLogger(logr.New(ctrlruntimelog.NullLogSink{}))
```

Comment:
1. Would it be better to put it in New() func?
2. We can use the klog instance to set the logger of the runtime-controller.
3. Do other runtime-controllers also need to set logger?
```golang
func NewAccessRoleControllerManager() {
    ...
    ctrlruntimelog.SetLogger(klog.Background())
}
```